assets/styles.css

```diff
@@ -0,0 +1,113 @@
+.grid{
+    width: 100vw;
+    height: 100vh;
+
+    display: grid;
+    grid-template: 200px 1fr 1fr 250px / 1fr 1fr 1fr;
+    
+}
+
+header{
+    background-color: #223E88;
+    grid-column-start: 1;
+    grid-column-end: 4;
+}
+
+.logo{
+    padding: 40px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+img{
+    max-width: 100px;
+ }
+
+h1{
+    text-align: center;
+    color: goldenrod;
+    font-family: serif;
+}
+
+main{
+    background-color: #D8D8D8;
+    grid-row-start: 2;
+    grid-column-start: 2;
+    grid-column-end: 3;
+}
+
+footer{
+    grid-row-start: 4;
+    grid-column-start:1;
+    grid-column-end: 4;
+}
+
+
+nav{
+    grid-row-start:2;
+    grid-column-start: 1;
+    margin:7.5px 0px 15px;
+  
+}
+
+.nav-wrapper{
+    margin: 5px 5px;
+    border: solid black 1px;
+    
+}
+
+body{
+    background-color: #D8D8D8;
+}
+
+form, .page-footer.page-footer{
+    background-color: #800507;
+}
+
+.card.card{
```

Comment:
Do we need a double .card.card or do you mean .card? I can't see any element which has "card card" thing as per [StackOverflow post](https://stackoverflow.com/questions/11918491/using-two-css-classes-on-one-element).